### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by GodOfProDev**
I changed it so it specifies that you need to have a fork of the repo to contribute.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ Ever since Moulberry has stopped working on NEU, other contributors have been wo
 
 ### Software configuration
 
-- Clone the NEU repository using `git clone https://github.com/NotEnoughUpdates/NotEnoughUpdates`.
+- Fork the NEU repository using the fork button on top right of the page.
+- Clone the forked repository using `git clone https://github.com/<YourUserName>/<ForkedRepoName>`.
```
Comment (hannibal002) [inline CONTRIBUTING.md:24]: Maybe specify that they should name the fork NotEnoughUpdates instead of having the placeholder ForkedRepoName?

Comment (hannibal002) [reply CONTRIBUTING.md:24]: Not that it's a requirement, but it reduces the possibility of some dev thinking he needs to name it something else to work.

**Review by hannibal002 — APPROVED** [review]

**Review by Lulonaut — APPROVED** [review]

**Review by Ascynx — APPROVED** [review]

**Review by Lulonaut — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -20,7 +20,11 @@ Ever since Moulberry has stopped working on NEU, other contributors have been wo
 
 ### Software configuration
 
-- Clone the NEU repository using `git clone https://github.com/NotEnoughUpdates/NotEnoughUpdates`.
+- Fork the NEU repository using the fork button on top right of the page and name the repo NotEnoughUpdates.
+- Clone the forked repository using `git clone https://github.com/<YourUserName>/NotEnoughUpdates`.
+- Make sure to create new branches for features you are working and not commit to the master branch of your repository
```
Comment (Lulonaut) [inline CONTRIBUTING.md:25]: actually, please be consistent with the dots at the end of sentences

Comment (GodOfProDev) [reply CONTRIBUTING.md:25]: Ok Let me do that

**Review by Lulonaut — APPROVED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -20,7 +20,11 @@ Ever since Moulberry has stopped working on NEU, other contributors have been wo
 
 ### Software configuration
 
-- Clone the NEU repository using `git clone https://github.com/NotEnoughUpdates/NotEnoughUpdates`.
+- Fork the NEU repository using the fork button on top right of the page and name the repo NotEnoughUpdates.
+- Clone the forked repository using `git clone https://github.com/<YourUserName>/NotEnoughUpdates`.
+- Make sure to create new branches for features you are working and not commit to the master branch of your repository.
+- After you have comitted all of the necessary changes make a pr on that branch.
```
Comment (walkerselby) [inline CONTRIBUTING.md:26]: "comitted" should be "committed"

Comment (walkerselby) [inline CONTRIBUTING.md:26]: If we are writing it this thoroughly, might as well write out "pull request" rather than pr

**Review by walkerselby — CHANGES_REQUESTED** [review]
Two minor things

File: CONTRIBUTING.md (hunk 3 above)
Comment (walkerselby) [reply CONTRIBUTING.md:26]: and add a comma after changes.

Comment (GodOfProDev) [reply CONTRIBUTING.md:26]: Ok

Comment (GodOfProDev) [reply CONTRIBUTING.md:26]: Will do that

**Review by walkerselby — APPROVED** [review]

**Review by Lulonaut — APPROVED** [review]

**Comment by GodOfProDev** [comment]
Can you remove that other request ? i did both of them in 1 commit and it marks it as unresolved

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -20,7 +20,11 @@ Ever since Moulberry has stopped working on NEU, other contributors have been wo
 
 ### Software configuration
 
-- Clone the NEU repository using `git clone https://github.com/NotEnoughUpdates/NotEnoughUpdates`.
+- Fork the NEU repository using the fork button on top right of the page and name the repo NotEnoughUpdates.
+- Clone the forked repository using `git clone https://github.com/<YourUserName>/NotEnoughUpdates`.
+- Make sure to create new branches for features you are working and not commit to the master branch of your repository.
```
Comment (jani270) [inline CONTRIBUTING.md:25]:
```suggestion
- Make sure to create new branches for features you are working on and not commit to the master branch of your repository.
```

**Review by jani270 — CHANGES_REQUESTED** [review]

**Review by jani270 — APPROVED** [review]
Looks fine now! 👍

**Review by walkerselby — APPROVED** [review]

**Review by Lulonaut — APPROVED** [review]
final review :flushed:

**Review by romangraef — APPROVED** [review]

**Pull request merged**